src/plugins/data_source_management/public/components/create_data_source_wizard/components/create_form/create_data_source_form.tsx

```diff
@@ -211,22 +213,29 @@ export class CreateDataSourceForm extends React.Component<
 
   /* Render header*/
   renderHeader = () => {
```

Comment:
Not related to this specific PR, but why are we still using class based react components here?

Reply:
Previously we had to do some validations as soon as state was changing. It was not possible in functional components. Also there were few more reasons. Now lot of things changed. 

I will discuss with team and make changes accordingly for next time

Reply:
Thanks for the context @mpabba3003 :) yeah i'd love to know what that scenario was.